gradle/ext.gradle

```diff
@@ -80,17 +80,16 @@ gradle.ext {
     component_type = component_type_default
 }
 
-if (System.env.containsKey("EXAMPLE_AS_ROOT")) {
-    gradle.ext.example_as_root = Boolean.parseBoolean(System.env.EXAMPLE_AS_ROOT)
-    logger.lifecycle "System env EXAMPLE_AS_ROOT set to $System.env.EXAMPLE_AS_ROOT, using $gradle.ext.example_as_root"
+if (System.env.containsKey("EXAMPLE_EMBEDDING_METHOD")) {
+    gradle.ext.example_embedding_method = System.env.EXAMPLE_EMBEDDING_METHOD
+    logger.lifecycle "System env EXAMPLE_EMBEDDING_METHOD set to $System.env.EXAMPLE_EMBEDDING_METHOD, using $gradle.ext.example_embedding_method"
 } else {
-    // load some more from local.properties or set defaults.
-    gradle.ext.example_as_root = exampleAsRootLocalProperties?.toBoolean() ?: false
-    logger.lifecycle "local.properties read (kaluga.exampleAsRoot=$exampleAsRootLocalProperties, using $gradle.ext.example_as_root)"
+    gradle.ext.example_embedding_method = exampleEmbeddingMethodLocalProperties ?: "composite"
+    logger.lifecycle "local.properties read (kaluga.exampleEmbeddingMethod=$exampleEmbeddingMethodLocalProperties, using $gradle.ext.example_embedding_method)"
 }
 
 if (System.env.containsKey("EXAMPLE_MAVEN_REPO")) {
-    gradle.ext.example_maven_repo = System.env.EXAMPLE_MAVEN_REPO
+    gradle.ext.example_maven_addIntoExamplerepo = System.env.EXAMPLE_MAVEN_REPO
```

Comment:
`example_maven_addIntoExampleRepo` also log below has to be updated for new variable name

Reply:
Or even `example_maven_add_into_example_repo`

Reply:
Don't know how this change sneaked into the PR 😅

Reply:
Fixed